backend/scripts/import-posts/map-airtable-data.js

```diff
@@ -0,0 +1,76 @@
+// https://stackoverflow.com/questions/52456065/how-to-format-and-validate-email-node-js/52456632
+const emailRegexp = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
+
+const cleanEmail = (email) => {
+  if (!email) return false;
+  // a few with multiple emails, only take first
+  let [cleaned] = email.split(",");
+  // trim trailing whitespace & periods
+  cleaned = cleaned.trim();
+  cleaned = cleaned.replace(/^\.|\.$/g, "");
+  // some other non-emails (extra link, whatsapp), just skip these
+  return emailRegexp.test(cleaned) && cleaned;
+};
+
+module.exports = (post, sourcedBy) => {
+  /* eslint-disable prefer-const */
+  let {
+    android_app: playStore,
+    city,
+    country,
+    from_whom: [authorType] = ["Other"], // provided as array even though we only want [one]
+    ios_app: appStore,
+    post_description: content,
+    post_email: email,
+    post_language: language,
+    post_name: title,
+    post_objective: [objective],
+    post_type: types,
+    post_website: website,
+    share_with: [visibility] = ["worldwide"],
+    state,
+  } = post.fields;
+  /* eslint-enable prefer-const */
+
+  if (authorType === "Others") {
+    authorType = "Other";
+  } else if (authorType === "Non-Profit") {
+    authorType = "Non-profit";
+  }
+  objective = objective.toLowerCase();
+  visibility = visibility === "Global" ? "worldwide" : visibility.toLowerCase();
```

Comment:
Most of these `.toLowerCase()` will be done at the model automatically (check if the attribute at `Post` model has `lowercase: true` for that). The same from `trim`.